Src/RE2.Native/RE2.Native.x64.vcxproj

```diff
@@ -64,4 +64,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Visual Studio decided to make this file match the other RE2 projects and remove the last line. No other changes.